src/bit_set.zig

```diff
@@ -190,6 +190,14 @@ pub fn IntegerBitSet(comptime size: u16) type {
             return @ctz(mask);
         }
 
+        /// Finds the index of the first unset bit.
+        /// If all bits are set, returns null.
+        pub fn findFirstUnset(self: Self) ?usize {
```

Comment:
Haha, lol, we have _exactly_ the same thing in TigerBeetle, where we use `initFull` because there's no ffu, only ffs, and that bloats the binary